tests/testProducer.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+# Copyright 2019 AstroLab Software
+# Author: Abhishek Chauhan
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import os
+import io
+import confluent_kafka
+import json
+import glob
+import time
+import fastavro
+from pprint import pprint as pp
+from typing import Iterable
+
+def read_avro_alerts(data_path: str) -> Iterable[dict]:
+    """ read avro alert files and return an interable 
+    with dicts of alert data"""
```

Comment:
Docstring needs to be completed (Parameters, Returns).

Reply:
fixed in 8bb8eef